# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/basetypes/Accessible.h
# Commit: 6591f14bbc71
# Full Hash: 6591f14bbc713f11b6316712d931ecc19a443a5c
# Author: Morgan Reschenberg <mreschenberg@mozilla.com>
# Date: 2021-10-05 03:52:32
# Regressor Bug: 1726227
# File Overlap Count: 1
# Description:
#   Bug 1726227: Cache parent-relative accessible bounds, resolution in parent process r=Jamie,eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123399
# ==============================================================================

diff -r 06dda9072cb7 -r 6591f14bbc71 accessible/basetypes/Accessible.h
--- a/accessible/basetypes/Accessible.h	Mon Oct 04 19:28:49 2021 +0000
+++ b/accessible/basetypes/Accessible.h	Mon Oct 04 19:46:54 2021 +0000
@@ -9,6 +9,7 @@
 #include "mozilla/a11y/Role.h"
 #include "mozilla/a11y/AccTypes.h"
 #include "nsString.h"
+#include "nsRect.h"
 
 class nsAtom;
 
@@ -125,6 +126,8 @@
   virtual double MaxValue() const = 0;
   virtual double Step() const = 0;
 
+  virtual nsIntRect Bounds() const = 0;
+
   // Type "is" methods
 
   bool IsDoc() const { return HasGenericType(eDocument); }